maintainers/maintainer-list.nix

```diff
@@ -6476,6 +6476,11 @@
     github = "icewind1991";
     githubId = 1283854;
   };
+  icyrockcom = {
+    github = "icyrockcom";
+    githubId = 785140;
```

Comment:
maybe add another contact method other than github

Reply:
Is another one required or is github good enough?

Reply:
Many people add email, etc., but they should not be required.

Reply:
OK if not required I'd leave it like this if that's OK with you.

Reply:
Thats okay but if you're github gets deleted, your maintainer entry will probably just be removed.

Reply:
> Thats okay but if you're github gets deleted, your maintainer entry will probably just be removed.

That's fine, thanks @SuperSandro2000.